src/test/java/org/openrewrite/hibernate/MigrateToHibernate61Test.java

```diff
@@ -155,5 +156,204 @@ public class TestApplication {
         );
     }
 
+    @Test
+    void hypersistenceUtilsTypesMigrated() {
+        rewriteRun(
+          spec -> spec.typeValidationOptions(TypeValidation.none()),
```

Comment:
In rare cases disabling type validation is the only option, but typically it is very helpful in finding any missing type attribution. In this case what should be added is this:

```suggestion
          spec -> spec.parser(JavaParser.fromJavaVersion()
                  .classpathFromResources(new InMemoryExecutionContext(), "hibernate-core-5.6", "javax.persistence-api-2.2")),
```

Please note that this requires you to add the following block to `build.gradle.kts`:

```kts
recipeDependencies {
    parserClasspath("org.hibernate:hibernate-core:5.+")
    parserClasspath("javax.persistence:javax.persistence-api:2.+")
}
```

Also, after adding that block make sure to run `./gradlew downloadRecipeDependencies` and then also adding the downloaded jar files to this commit.

Reply:
Fixed just now after an initial classpath entry was added in #8